gapis/server/server.go

```diff
@@ -113,29 +113,37 @@ func (s *server) GetServerInfo(ctx context.Context) (*service.ServerInfo, error)
 	return s.info, nil
 }
 
-func (s *server) CheckForUpdates(ctx context.Context, includePrereleases bool) (*service.Release, error) {
+func (s *server) CheckForUpdates(ctx context.Context, includeDevReleases bool) (*service.Release, error) {
 	const (
-		githubOrg  = "google"
-		githubRepo = "gapid"
+		githubOrg     = "google"
+		githubRepo    = "gapid"
+		devGithubRepo = "gapid-dev-releases"
 	)
 	ctx = status.Start(ctx, "RPC CheckForUpdates")
 	defer status.Finish(ctx)
 	ctx = log.Enter(ctx, "CheckForUpdates")
-	client := github.NewClient(nil)
+
 	options := &github.ListOptions{}
+	client := github.NewClient(nil)
 	releases, _, err := client.Repositories.ListReleases(ctx, githubOrg, githubRepo, options)
 	if err != nil {
 		return nil, log.Err(ctx, err, "Failed to list releases")
 	}
+
+	if includeDevReleases {
+		devClient := github.NewClient(nil)
```

Comment:
Is it not possible to use the same client?

Reply:
Yes indeed, fixed.